.github/workflows/test.sh

```diff
@@ -14,3 +14,17 @@ for user in ${users[@]}; do
         exit 1
     fi
 done
+
+echo "Parsing caat.json"
+users="$(jq -r .userMap[][] caat.json)"
```

Comment:
You can just add caat.json to the above `jq` call as an additional parameter, instead of copying the whole code block.